### DISCUSSION TITLE
[Question] Error Resolving Module "@adraffy/ens-normalize" with React Native 0.68 and web3modal Integration

### DISCUSSION THREAD

**Discussion by judeProground** [WalletConnect/walletconnect-monorepo#4003] [Developer Support]
Hello, is there a minimum version of react-native required to integrate `@web3modal/wagmi-react-native`? I'm encountering an error, specifically 'While trying to resolve module @adraffy/ens-normalize ... ' at runtime after installing the web3modal packages with React Native version 0.68. (`@adraffy/ens-normalize` is dependency of `viem`)

**Comment by glitch-txs** [comment]
hi could you open an issue in the RN repository also with a minimal reproducible example? Thanks!
